src/c3chart/Common.js

```diff
@@ -21,6 +22,10 @@
             data: {}
         };
         this._prevColumnIDs = [];
+        
+        this._config.data.onclick = function (d, element) {
+            context.click(context.rowToObj(context.data()[d.index]), d.id, context.c3Chart.selected().length > 0);
```

Comment:
The click event should be (row, col, selected) you are now omitting the col info...